crates/node/engine/src/metrics/mod.rs

```diff
@@ -0,0 +1,36 @@
+//! Metrics for the engine
+
+/// Container for metrics.
+#[derive(Debug, Clone)]
+pub struct Metrics;
+
+impl Metrics {
+    /// Identifier for the gauge that tracks block labels.
+    pub const BLOCK_LABELS: &str = "kona_node_block_labels";
+
+    /// Initializes metrics for the engine.
+    ///
+    /// This does two things:
+    /// * Describes various metrics.
+    /// * Initializes metrics to 0 so they can be queried immediately.
+    pub fn init() {
+        Self::describe();
+        Self::zero();
+    }
+
+    /// Describes metrics used in [`kona_engine`][crate].
+    pub fn describe() {
+        metrics::describe_gauge!(Self::BLOCK_LABELS, "Blockchain head labels");
+    }
+
+    /// Initializes metrics to `0` so they can be queried immediately by consumers of prometheus
+    /// metrics.
+    pub fn zero() {
+        // Blockchain head labels
```

Comment:
can you explain why you chose to use gauge here pls? just curious

Reply:
Counter would be the proper one to use, but `kona-macros` doesn't look to support counters well atm. We can follow up with moving over to counters with some added support there if we want.